content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,439 @@
+
+## Contact Configuration
+
+Configuring a contact requires you to define the name of the contact and the
+services plus configuration override for that contact.
+
+{{< highlight json >}}
+{
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In the example above we have a contact named **support**. For **support** we
+have an override configuration for email and slack. In our example we're having
+this contact email _support@sensuapp.com_ instead of the email intergration's
+default **to** address. For Slack, we're defining the channel the event should
+post to, in this case _#support_.
+
+## Check Configuration
+
+Once a contact has been defined, we can now apply which contact to use in a
+check configuration. The use case for having it on a check could be that the
+client is owned by a particular team or group, the same can be done for specific
+checks. For instance 
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handler": "email",
+      "contact": "support"
+    }
+  }
+}
+{{< /highlight >}}
+
+## Client Configuration
+
+A client configuration can include a contact to use.
+
+{{< highlight json >}}
+{
+  "client": {
+    "name": "ec2-west-id-12345",
+    "address": "8.8.8.8",
+    "subscriptions": [
+      "production",
+      "aws"
+    ],
+    "contact": "aws-team"
+  }
+}
+{{< /highlight >}}
+
+# Example Implementations
+
+## Default Configuration
+
+For the examples we'll be working with we want to define what our default handlers will be.
+
+In the following section we'll be going over different scenarios and showcase how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and Pagerduty as our example handlers.
+
+Bellow is the default configuration for the handlers we'll be working with in our examples:
+
+{{< highlight json >}}
+{
+  "handlers": {
+    "slack": {
+      "webhook_url": "https://hooks.slack.com/services/foo/bar/foobar",
+      "username": "sensu",
+      "channel": "#alerts",
+      "timeout": 10
+    },
+    "email": {
+      "smtp": {
+        "address": "smtp.example.com",
+        "port": 587,
+        "openssl_verify_mode": "none",
+        "enable_starttls_auto": true,
+        "authentication": "plain",
+        "user_name": "postmaster@example.com",
+        "password": "SECRET"
+      },
+      "to": "support@example.com",
+      "from": "noreply@example.com",
+      "timeout": 10
+    },
+    "pagerduty": {
+      "service_key": "someservicekey ",
+      "timeout": 10
+    }
+  }
+}
+{{< /highlight >}}
+
+## Multiple Handlers With Matching Contacts
+
+TODO: Check with email and slack handler, contact with email and slack
+
+In this configuration we see that our contact and check both have a
+configuration for email and slack.
```

Comment:
It might be worth it to be more explicit here since it's not the same type of configuration on the check and the contact.